### ISSUE TITLE
Freezing on startup

### ISSUE THREAD

**Issue by clmisback** [LorenzCK/OnTopReplica#160]
Version: 3.5.1 (latest at time of post)

Windows Version: Windows 11 Pro 23H2

Everytime I startup the app it freezes for ~10 seconds. After, everything works as expected. I feel like it started when I updated to windows 11, but I don't know for sure when it started. Uninstalled and reinstalled the latest version and the same freeze occurs.

btw love the app, I use it all the time. Thanks :)

Does anyone have this issue?

**Comment by VackerSimon** [comment]
@clmisback I have not had that issue and I'm also on Windows 11. Have you tried deleting the config located in `%USERPROFILE%\AppData\Local\Lorenz_Cuno_Klopfenstein`?

**Comment by clmisback** [comment]
I tried that but it was still freezing.

Interestingly enough, I thought it might be something with other apps that I'm running, so I closed them one by one and tried starting OnTopReplica each time.

Turns out it's "Windscribe", my vpn. If the vpn is connected OnTopReplica freezes for on startup. When I turn off the vpn, no freeze and everything works as expected.

At this point I'm not sure if this is really the place for this issue but if anyone has any insight I'd appreciate it.

**Comment by LorenzCK** [comment]
Strange! The only thing that comes to mind is the auto-update check that is performed when OnTopReplica is launched: in theory it should run in background, but maybe it does conflict with the VPN somehow and blocks the UI thread until the update check is done. 🤔

**Comment by clmisback** [comment]
Yeah, that seems to be it. If my VPN is connected and I click the "Check now!" button in the about window (under Updates), it will freeze for ~10 seconds then come up with "Failed to download update info".

![image](https://github.com/user-attachments/assets/cc64cf5c-f9c3-4aac-a695-afa31de33498)

If I do it with my VPN disconnected there is no freeze but it still comes up with the same "Failed to download update info" error.
